homeassistant/components/sleepiq/services.yaml

```diff
@@ -0,0 +1,22 @@
+set_sleep_number:
+  description: Changes the Sleep Number of a bed.
+  fields:
+    sleep_number:
+      description: The new sleep number, as a multiple of 5 between 5 and 100.
```

Comment:
Please update the service descriptions to match or latest standards: 

<https://developers.home-assistant.io/docs/dev_101_services#service-descriptions>